src/validate.js

```diff
@@ -0,0 +1,57 @@
+import fs from "fs"
+import path from "path"
+import glob from "glob"
+import { diff } from "just-diff"
+
+export default () => {
+  // todo: test both in project because of the path here...
```

Comment:
That glob looks reasonable to me. do we still need the TODO?

Reply:
yeah nope that's a leftover. I wanted to check whether the glob path refers to the project directory when called from the project and refers to the package directory when ran from the package